starter/src/app/app.module.ts

```diff
@@ -47,6 +52,9 @@ import { NoContentComponent } from "./no-content";
 import { XLargeDirective } from "./home/x-large";
 import { DevModuleModule } from "./+dev-module";
 /* tslint:disable:no-import-side-effect */
+// load PCSS styles
+import "../styles/styles.pcss";
```

Comment:
Do we really need to import styles.pcss here ?

Reply:
Yes, we need to import it so that webpack can transform these styles.

In fact this is the same we were doing in Stark 9 , we loaded them inside the main.ts in PROD and main-styles in DEV ;)